docs/tutorials/projects/tipsy.md

```diff
@@ -323,6 +318,7 @@ Click on the **Services** subtab.
 
 1. **Configure the ML model service**
 
+   The [ML model service](/ml/) allows you to deploy a machine learning model to your robot.
```

Comment:
No-op: Leaving `robot` because `machine learning model to machine` and because later UI element **Deploy model on robot**